content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -684,13 +746,21 @@ Here are a few things to check for:
 - **Is the auto-instrumentation for the right language?** For example, when
   instrumenting a Python application, make sure that the annotation doesn't
   incorrectly say `instrumentation.opentelemetry.io/inject-java: "true"`
-  instead.
+    instead.
 - **Is the auto-instrumentation annotation in the correct location?** When
   defining a `Deployment`, annotations can be added in one of two locations:
   `spec.metadata.annotations`, and `spec.template.metadata.annotations`. The
   auto-instrumentation annotation needs to be added to
   `spec.template.metadata.annotations`, otherwise it won’t work.
 
+{{% alert title="Note" color="info" %}}
+
+Note that unlike other auto-instrumentation, the Deno auto-instrumentation uses
+the `instrumentation.opentelemetry.io/inject-sdk: "true"` annotation, not an
+annotation that contains the string `deno`.
+
+{{% /alert %}}
+
```

Comment:
I'd rather drop this alert and fold this alert text into a sub-bullet of **Is the auto-instrumentation for the right language?**, which already mentions exceptions (currently for Python). For example, consider replacing the bullet at line 746 of this PR (line 684 in the original file), with:

```markdown
- **Is the auto-instrumentation for the right language?** For example, when instrumenting an application in:
  - Deno, which uses the `instrumentation.opentelemetry.io/inject-sdk: "true"` annotation, rather than an annotation containing the string `deno`.
  - Python: ensure that the annotation doesn't incorrectly say `instrumentation.opentelemetry.io/inject-java: "true"` instead.
```

@open-telemetry/docs-approvers WDYT?

Reply:
I added it to that bullet, but not quite like you suggested - the Python comments seems more general "for example, if you use python, don't use the java annotation" - it isn't: "specifically if you use python, do not use the java annotation".

LMK if the new wording works for you.

Reply:
Yes, that is good. The only adjustment that I'd like to ask is that you make those two sentences subbullets so that the Deno statement is easier to see -- I might even suggest that you put **Deno** in bold there, since it is in effect an important "Note".

Reply:
adjusted.